upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -1392,25 +1392,6 @@ func (n *nodeUpConfigBuilder) BuildConfig(ig *kops.InstanceGroup, apiserverAddit
 		}
 	}
 
-	useConfigServer := featureflag.KopsControllerStateStore.Enabled() && (role != kops.InstanceGroupRoleMaster)
```

Comment:
Ah, this is nice, I like the idea of using the helper function instead of the flag :-)